Lib/glyphsLib/types.py

```diff
@@ -228,7 +228,12 @@ def read(self, src):
         """Parse a datetime object from a string."""
         # parse timezone ourselves, since %z is not always supported
         # see: http://bugs.python.org/issue6641
-        string, tz = src.rsplit(' ', 1)
+        srcs = src.split(' ')
+        string = srcs[0] + " " + srcs[1]
+        if len(srcs) > 2:
+            tz = srcs[2]
```

Comment:
you're assuming that the third item in the list is the time zone. Couldn't it also be the optional 'AM'/'PM' string?

Reply:
That was already like this. I just adapted it.

Reply:
but the tests are failing now because of that
https://travis-ci.org/googlei18n/glyphsLib/jobs/315772812#L756